src/main/kotlin/org/jitsi/nlj/stats/NodeStatsBlock.kt

```diff
@@ -57,4 +115,23 @@ class NodeStatsBlock(val name: String) {
             toString()
         }
     }
-}
\ No newline at end of file
+
+    /**
+     * Returns a JSON representation of this [NodeStatsBlock].
+     */
+    fun toJson(): JSONObject = JSONObject().apply {
```

Comment:
Wondering if we should just do this elsewhere so this class doesn't get wrapped up in handling the formats we want this data in and just exposes it in its own format for others to translate as they want. I don't think this does anything special that requires this logic to be a member, does it?

Reply:
It could even be an extension function to this class defined elsewhere

Reply:
No, there's nothing that requires this to be a member. An extension function would be less convenient to use in Java, though (e.g. https://github.com/jitsi/jitsi-videobridge/pull/814/files#diff-183c330760fe6cb724a11713844ae382R1321)